docs/simbad/query_tap.rst

```diff
@@ -0,0 +1,274 @@
+
+Available columns
+^^^^^^^^^^^^^^^^^
+
+`~astroquery.simbad.SimbadClass.columns` lists the columns in all or a subset of SIMBAD tables.
+Calling it with no argument returns the 289 columns of SIMBAD. To restrict the output to
+some tables, add their name. To get the columns of the table ``ref``:
+
+.. doctest-remote-data::
+
+    >>> from astroquery.simbad import Simbad
+    >>> Simbad.columns("ref")
+    <Table length=11>
+    table_name column_name   datatype  ...  unit          ucd         
+      object      object      object   ... object        object       
+    ---------- ----------- ----------- ... ------ --------------------
+           ref    abstract UNICODECHAR ...                 meta.record
+           ref     bibcode        CHAR ...            meta.bib.bibcode
+           ref         doi     VARCHAR ...          meta.code;meta.bib
+           ref     journal     VARCHAR ...            meta.bib.journal
+           ref   last_page     INTEGER ...               meta.bib.page
+           ref    nbobject     INTEGER ...                 meta.number
+           ref      oidbib      BIGINT ...        meta.record;meta.bib
+           ref        page     INTEGER ...               meta.bib.page
+           ref       title        CLOB ...                  meta.title
+           ref      volume     INTEGER ...             meta.bib.volume
+           ref        year    SMALLINT ...          meta.note;meta.bib
+
+`~astroquery.simbad.SimbadClass.find_columns_by_keyword` returns columns for witch the 
+given keyword is either in the table name, in the column name or in its description.
+This is not case-sensitive.
+
+.. doctest-remote-data::
+
+    >>> from astroquery.simbad import Simbad
+    >>> Simbad.find_columns_by_keyword("Radial velocity")
+    <Table length=8>
+      table_name    column_name   ...  unit                  ucd                
+        object         object     ... object                object              
+    ------------- --------------- ... ------ -----------------------------------
+            basic     rvz_bibcode ...        meta.bib.bibcode;spect.dopplerVeloc
+            basic         rvz_err ... km.s-1       stat.error;spect.dopplerVeloc
+            basic    rvz_err_prec ...                                           
+            basic        rvz_qual ...          meta.code.qual;spect.dopplerVeloc
+            basic      rvz_radvel ... km.s-1              spect.dopplerVeloc.opt
+            basic rvz_radvel_prec ...                                           
+            basic        rvz_type ...                                           
+    mesVelocities          origin ...                                  meta.note
+
+Example TAP queries
+^^^^^^^^^^^^^^^^^^^
+
+This section lists more complex queries by looking at use cases from former astroquery issues.
+
+`Getting all bibcodes containing a certain type of measurement for a given object. <https://github.com/astropy/astroquery/issues/243>`__
```

Comment:
No need to link to github issues, remove the link from the heading here and below, too

Reply:
Done :)